packages/hardhat-chai-matchers/test/reverted/reverted.ts

```diff
@@ -245,7 +245,8 @@ describe("INTEGRATION: Reverted", function () {
     });
 
     describe("calling a method that reverts without a reason", function () {
-      it("successful asserts", async function () {
+      // depends on a bug being fixed on ethers.js
```

Comment:
can we cite the ethers Issue here? by "being fixed" do you mean that it's being actively worked on, and expected to be fixed relatively soon? or that we should skip these tests indefinitely? (or something in between?)

Reply:
There's no issue yet, I asked Richard about this but he hasn't responded yet